frontend/package.json

```diff
@@ -103,8 +103,8 @@
     "whatwg-fetch": "3.6.20"
   },
   "engines": {
-    "node": "v18.17.1",
-    "pnpm": "v9.11.0"
+    "node": "18.17.1",
+    "pnpm": "9.15.0"
```

Comment:
My interpretation of the docs https://docs.npmjs.com/cli/v10/configuring-npm/package-json#engines is that this should not contain a tag but a version

It seems that some tools read this correctly, but dependabot,for example, doesn't